client/config.go

```diff
@@ -148,32 +150,39 @@ func (c Configuration) NewAdminClient(
 	writeRequestScope := iopts.MetricsScope().SubScope("write-req")
 	fetchRequestScope := iopts.MetricsScope().SubScope("fetch-req")
 
-	topoInit := params.TopologyInitializer
-	if topoInit == nil {
-		if c.ConfigService == nil {
-			return nil, errConfigurationMustSupplyConfigService
-		}
+	configureResults := &environment.ConfigureResults{
```

Comment:
nit: Call this `envCfg` so that when reading the result it seems pragmatic, i.e. `envCfg.NamespaceInitializer`